packages/ui5-app/ui5.yaml

```diff
@@ -68,6 +70,13 @@ server:
       transpileAsync: true
       excludePatterns:
       - "lib/"
+  - name: ui5-middleware-stringreplacer
+    afterMiddleware: compression
+    configuration:
+      debug: true
+      files:
+        - "**/*.js"
+        - "**/*.xml"
```

Comment:
can you please add some replacement configuration as described in the task's/middleware's README?

Reply:
Ack